JavaScript/JavaScriptSDK/Sender.ts

```diff
@@ -168,22 +180,17 @@ module Microsoft.ApplicationInsights {
                 // Send data only if disableTelemetry is false
                 if (!this._config.disableTelemetry()) {
```

Comment:
I think now I might have read the condition backwards, maybe I missed the ! character or something.  Sorry. :)